cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -335,9 +358,10 @@ def _cleanup():
     required_flags=["v2v"],
     scope="module",
 )
-def test_v2v_rbac(appliance, new_credential):
+def test_v2v_rbac(appliance, setup_user_for_v2v_migration):
     """
-    Test migration with role-based access control
+    Test to verify that the Migration tab is available/unavailable in the UI with
+    role-based access control
 
     Polarion:
         assignee: sshveta
```

Comment:
Please change the assignee to yourself

Reply:
Done.

Also, parametrized the test on 'migration_feature_availability_for_role'.